model/messaging/deprecated/registry-deprecated.yaml

```diff
@@ -49,10 +49,10 @@ groups:
       - id: messaging.servicebus.destination.subscription_name
         type: string
         brief: >
-          Deprecated, use `messaging.servicebus.destination.subscription_name` instead.
+          Deprecated, use `messaging.destination.subscription.name` instead.
```

Comment:
Should we stop modifying both `brief` and `deprecated` and instead update markdown gen to fill in deprecated language for brief?

Reply:
Yes that would be great. A step towards make it standarized!